File: app3/src/nl/ruuda/mindec/MainActivity.java
package nl.ruuda.mindec;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends Activity
{
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        TextView label = new TextView(this);
        label.setText("Mindec");

        setContentView(label);
    }
}

File: app3/src/main/MainActivity.kt
package nl.ruuda.mindec;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState);

        val label = TextView(this);
        label.setText("Mindec");

        setContentView(label);
    }
}

